website/package.json

```diff
@@ -8,7 +8,7 @@
     "npm": ">=7.0.0"
   },
   "devDependencies": {
-    "@hashicorp/platform-cli": "^1.2.0",
+    "@hashicorp/platform-cli": "^2.5.1",
```

Comment:
This breaking upgrade is considered safe. See discussion on a [related PR in vagrant. ](https://github.com/hashicorp/vagrant/pull/13059#pullrequestreview-1252499799)